examples/lib/src/main/java/momento/client/example/DictionaryExample.java

```diff
@@ -21,25 +20,15 @@
 import org.slf4j.LoggerFactory;
 
 public class DictionaryExample {
-
-  private static final String AUTH_TOKEN_ENV_VAR = "MOMENTO_AUTH_TOKEN";
   private static final Duration DEFAULT_ITEM_TTL = Duration.ofSeconds(60);
-
   private static final String CACHE_NAME = "dictionary-example-cache";
   private static final String DICTIONARY_NAME = "example-dictionary";
-
   private static final Logger logger = LoggerFactory.getLogger(DictionaryExample.class);
 
   public static void main(String[] args) {
     logStartBanner(logger);
 
-    final CredentialProvider credentialProvider;
-    try {
-      credentialProvider = new EnvVarCredentialProvider(AUTH_TOKEN_ENV_VAR);
-    } catch (SdkException e) {
-      logger.error("Unable to load credential from environment variable " + AUTH_TOKEN_ENV_VAR, e);
-      throw e;
-    }
```

Comment:
Thanks for the PR!

If this was production code this would be an awesome change. However, for the stuff in these example files, we are expecting that users may want to copy/paste code from one of these files directly into their app. So we are trying to avoid them having to navigate through multiple files in the project in their IDE. Or like, if they didn't clone the repo and they are just looking at the code on github, then they'd have to figure out where the util code lived and open two browser tabs to copy all the code they need.

So I think for examples we probably have a preference for "self-sufficient" over DRY. Let me know if you have different thoughts!

Reply:
No that makes total sense. I did see this class called ```ExampleUtils``` class which had the utility methods of printing the start and end banners so thought it'd be okay. I will like it to be consistent and get rid of that util as well in that case and make every example class it's own standalone with no helpers. We can also rework on this PR https://github.com/momentohq/client-sdk-java/pull/294/files and have a separate example altogether that uses secrets manager. 

Let me know if that makes sense and I can abandon these two PRs and create a fresh one that's more like ```BasicExampleWithSecretsManager``` in the examples directory; and also remove any util classes even if it's for the banner.

Reply:
yep! you nailed it. What happened with that ExampleUtils class is that in Nate's original PR where he created it, he had done exactly what you did with the auth, and I made this same comment. So he unrolled the auth stuff but left the banners, which I didn't care as much about. Personally I have no objection to you rolling those back too.

And yes, we should set up a separate example project to house the secrets manager thing :) that way the gradle build for these basic examples doesn't have a dependency on secrets manager, in case people are copying our gradle project for the examples. We'll need to break up the examples dir into some subdirs for different kinds of examples, which is a thing we did very recently for the node.js SDK and probably needs to happen to all of the SDKs over time. :)